Provis.WebApi/Controllers/WorkspaceController.cs

```diff
@@ -104,5 +104,16 @@ public async Task<IActionResult> GetWorkspaceMembersAsync(int workspaceId)
             var members = await _workspaceService.GetWorkspaceMembersAsync(workspaceId);
             return Ok(members);
         }
+
+        [Authorize]
+        [HttpDelete]
+        [WorkspaceRoles(new WorkSpaceRoles[] { WorkSpaceRoles.OwnerId })]
+        [Route("del/{workspaceId}/user/{userId}")]
```

Comment:
it isn't necessary write "del" in route because you use delete http method

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/93646070/147972820-650ca528-ff34-46be-b6ef-67872b586a46.png)
Because i dont want to have Route with only "/Workspace

Reply:
Please remove this here and on client side